cmd/immuadmin/command/database.go

```diff
@@ -37,6 +39,8 @@ func addDbUpdateFlags(c *cobra.Command) {
 	c.Flags().String("replication-follower-username", "", "set username used for replication")
 	c.Flags().String("replication-follower-password", "", "set password used for replication")
 	c.Flags().Uint32("write-tx-header-version", 1, "set write tx header version (use 0 for compatibility with immudb 1.1, 1 for immudb 1.2+)")
+	c.Flags().Uint32("max-commit-concurrency", store.DefaultMaxConcurrency, "set the maximum commit concurrency")
```

Comment:
To discuss:

I called this `max-commit-concurrency` since currently that is what this parameter is doing inside store.

Reply:
we might even change the internal naming as well

Reply:
I'd leave it for future changes - this also affects names in the GRPC protocol